notes/v1.0.0.md

```diff
@@ -0,0 +1,214 @@
+
+### New APIs
+
+#### Moved all APIs from ecctl to the cloud-sdk-go
+
+Packages previously published a public API through `ecctl/pkg/...`, have
+been move to the `cloud-sdk-go`, with an `api` suffix in the package name.
+
+The full list of packages which have been added as part of this change is:
+
+* [`pkg/api/platformapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi)
+* [`pkg/api/platformapi/configurationtemplateapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/configurationtemplateapi)
+* [`pkg/api/platformapi/allocatorapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/allocatorapi)
+* [`pkg/api/platformapi/instanceconfigapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/instanceconfigapi)
+* [`pkg/api/platformapi/snaprepoapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/snaprepoapi)
+* [`pkg/api/platformapi/enrollmenttokenapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/enrollmenttokenapi)
+* [`pkg/api/platformapi/proxyapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/proxyapi)
+* [`pkg/api/platformapi/proxyapi/filteredgroupapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/proxyapi/filteredgroupapi)
+* [`pkg/api/platformapi/roleapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/roleapi)
+* [`pkg/api/platformapi/runnerapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/runnerapi)
+* [`pkg/api/platformapi/constructorapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/platformapi/constructorapi)
+* [`pkg/api/stackapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/stackapi)
+* [`pkg/api/userapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/userapi)
+* [`pkg/api/userapi/authapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/userapi/authapi)
+* [`pkg/api/userapi/authapi/adminapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/userapi/authapi/adminapi)
+* [`pkg/api/deploymentapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi)
+* [`pkg/api/deploymentapi/eskeystoreapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi/eskeystoreapi)
+* [`pkg/api/deploymentapi/noteapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi/noteapi)
+* [`pkg/api/deploymentapi/depresourceapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi/depresourceapi)
+* [`pkg/api/deploymentapi/deptemplateapi`](https://pkg.go.dev/github.com/elastic/cloud-sdk-go/pkg/api/deploymentapi/deptemplateapi)
+
+#### New deptemplateapi package
+
+Added all the APIs for the new deployment templates API which present the
+deployment templates in the "deployment" format definitions. Which are
+closer to the deployment create and update request payloads.
```

Comment:
```suggestion
deployment templates in the "deployment" format definitions. These APIs
use consume and produce structures which are closer to the deployment 
create and update request payloads.
```

Reply:
I kind of want to remove that last sentence all-together as people reading this won't know about the old deployment-template endpoints so it might sound confusing to say something is "closer to...". As in, closer than what?

Reply:
I think that makes sense, I'll remove that line